trunk/SUAVE/Methods/Aerodynamics/AVL/Data/Wing.py

```diff
@@ -79,11 +80,11 @@ def append_control_surface(self,control):
 class Control_Surface(Data):
```

Comment:
Should this build off the Control Surface component? Also maybe it should not be in the methods folder.